lib/portage/glsa.py

```diff
@@ -186,14 +184,11 @@ def getText(node, format, textfd=None):  # pylint: disable=redefined-builtin
     @return:	the (formatted) content of the node and its subnodes
                     except if textfd was not none
     """
-    if not textfd:
-        textfd = StringIO()
-        returnNone = False
-    else:
-        returnNone = True
-    if format in ["strip", "keep"]:
-        if node.nodeName in ["uri", "mail"]:
-            textfd.write(node.childNodes[0].data + ": " + node.getAttribute("link"))
+    if textfd:
+        return None
```

Comment:
As bizarre as this code is, I don't think we can do this because of:

````python
                raise GlsaFormatException(_("Invalid Tag found: "), subnode.nodeName)
````

I think we still have to process the input so we can throw an exception if the input is bad.